src/input.js

```diff
@@ -105,6 +107,7 @@ class Input extends Component {
         aria-expanded={isOpen}
         autoComplete="off"
         value={(inputValue === null ? selectedItemValue : inputValue) || ''}
+        autoFocus={this.menu}
```

Comment:
Gotcha. Do you mind wrapping it in `Boolean(this.menu)`? I just prefer to avoid type coercion in certain scenarios (this being one of them) :smile: